fecgov/fec-cms
# Upgrade karma to 4.4.1 and puppeteer 19.1.2

## Summary (required)

Fixes an issue with chromeheadless browser failing to start

### Required reviewers

1 developer

## Impacted areas of the application

General components of the application that this PR will affect:

-  Karma chromeheadless browser

## How to test

- Re-run [this branch](https://app.circleci.com/pipelines/github/fecgov/fec-cms/2244/workflows/e1946fbb-1180-45c5-b948-665288075d64/jobs/5582) in circleCI, make sure that it passes.
